### PR TITLE
documentation: update release notes and changelog

### PR DESCRIPTION
**Pull request by iGormilhit**
Updates the files for the v0.7.0 release.

* Update the RELEASE-NOTES.rst file.
* Update the CHANGES.rst file.

Co-Authored-by: Igor Milhit <igor.milhit@rero.ch>

## Why are you opening this PR?

- Which task/US does it implement?
- Which issue does it fix?

## How to test?

- What command should I have to run to test your PR?
- What should I test through the UI?

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Extracted translations?

### CODE REVIEW

File: RELEASE-NOTES.rst [hunk 1]
```diff
@@ -18,6 +18,226 @@
...
+v0.7.0
+------
+
+User interface
+~~~~~~~~~~~~~~
+
+-  Moves to ``rero-ils-ui`` version 0.1.0. ``rero-ils-ui`` is the
+   angular project for part of user interface (public search and
+   professional interface).
+-  Fixes in the source code some issues identified through the
```
Comment (mmo) [inline RELEASE-NOTES.rst:30]: Fixes some issues in the source code, identified through the

File: RELEASE-NOTES.rst [hunk 2]
```diff
@@ -18,6 +18,226 @@
...
+   translation process (in both projects: ``rero-ils`` and
+   ``rero-ils-ui``).
+
+Public interface
+^^^^^^^^^^^^^^^^
+
+-  Improves the request deletion by patron, to keep the request tab
```
Comment (mmo) [inline RELEASE-NOTES.rst:37]: Improves request deletion by a patron

File: RELEASE-NOTES.rst [hunk 3]
```diff
@@ -18,6 +18,226 @@
...
+   active after deletion.
+-  Fixes the cover image display of the document detailed and brief view
```
Comment (mmo) [inline RELEASE-NOTES.rst:39]: in the document detailed and brief views

File: RELEASE-NOTES.rst [hunk 4]
```diff
@@ -18,6 +18,226 @@
...
+   of the public interface.
+-  Displays the document title field in brief and detailed views.
+-  Adapts the configuration of the search views to be compatible with
+   ``ng-core``, in order to ensure that professional action button do
```
Comment (mmo) [inline RELEASE-NOTES.rst:43]: professional action buttons

File: RELEASE-NOTES.rst [hunk 5]
```diff
@@ -18,6 +18,226 @@
...
+
+Professional interface
+^^^^^^^^^^^^^^^^^^^^^^
+
+-  Adds examples in the placeholders in the patron editor.
+-  Adds a request button on the document detailed view, that allows a
+   librarian to place a request on an item on behalf of a patron. The
+   button opens a modal in which the librarian can scan a patron barcode
+   and select a pickup location.
+-  Truncates the document detailed view abstract and adds a *show more*
```
Comment (mmo) [inline RELEASE-NOTES.rst:54]: Truncates the abstract in the document detailed view

File: RELEASE-NOTES.rst [hunk 6]
```diff
@@ -18,6 +18,226 @@
...
+-  Displays the electronic location on the document detailed view.
+-  Displays the document title field in brief and detailed views.
+-  Improves the edition statement field display on the document detailed
+   view.
+-  Implements the fee tab of the patron account from the librarian point
+   of view.
+-  Fixes uniqueness value check of several fields in the location
+   record, such as ``is_online``, ``pickup_name``, etc., when a location
+   is created or updated.
+-  Adds guards to protect access of any resource editor based on the
```
Comment (mmo) [inline RELEASE-NOTES.rst:71]: access to

File: RELEASE-NOTES.rst [hunk 7]
```diff
@@ -18,6 +18,226 @@
...
+   user permissions. A basic error page is also added.
+-  Removes the location from the library brief view (search result) to
```
Comment (mmo) [inline RELEASE-NOTES.rst:73]: and moves it to

File: RELEASE-NOTES.rst [hunk 8]
```diff
@@ -18,6 +18,226 @@
...
+   the library detailed view.
+-  Hides action button on the location detailed view depending on the
```
Comment (mmo) [inline RELEASE-NOTES.rst:75]: Hides the action button

File: RELEASE-NOTES.rst [hunk 9]
```diff
@@ -18,6 +18,226 @@
...
+   user permission on the library (the parent record).
+
+Circulation
+~~~~~~~~~~~
+
+-  Generates fees (“patron transactions”) and “patron transaction
+   events” when an overdue occurs.
```
Comment (mmo) [inline RELEASE-NOTES.rst:82]: when a loan is overdue

File: RELEASE-NOTES.rst [hunk 10]
```diff
@@ -18,6 +18,226 @@
...
+
+-  Adds a new resource, “patron transaction”, to manage all the
+   different fees that a parton will generate (overdue checkout,
+   photocopy, subscription, lost or damaged item, interlibrary loan,
+   etc.)
+-  Adds a new resource, “patron transaction event” to track the history
+   of each fee (partially paid, paid, disputed, etc.)
+-  Adapts the “fee” resource to make use of “patron transaction” and
+   “patron transaction event” resource.
+-  Allow the librarian to register payment or partial payement for fees.
```
Comment (mmo) [inline RELEASE-NOTES.rst:97]: Allows the librarian

File: RELEASE-NOTES.rst [hunk 11]
```diff
@@ -18,6 +18,226 @@
...
+
+Metadata
+~~~~~~~~
+
+-  Implements the ``electronicLocator`` fields in the metadata model.
+-  Implements the ``title`` fields in the metadata model, with transformation
+   from RERO MARC21 and BNF UNIMARC, which was a huge work.
+-  Improves the ``marc21json`` CLI to enable debugging option.
```
Comment (mmo) [inline RELEASE-NOTES.rst:105]: options

File: RELEASE-NOTES.rst [hunk 12]
```diff
@@ -18,6 +18,226 @@
...
+-  Updates the location JSON schema to make the ``pickup_name`` field
+   required if the location is a pickup location (``is_pickup`` is set
+   to ``true``).
+-  Updates the documents records to the new MEF IDs.
```
Comment (mmo) [inline RELEASE-NOTES.rst:109]: document records :-)

File: RELEASE-NOTES.rst [hunk 13]
```diff
@@ -18,6 +18,226 @@
...
+-  Checks and removes leading or trailing spaces in the item and patron
+   barcodes.
+
+Acquisition
+~~~~~~~~~~~
+
+-  Adds “invoice” resource. This resource is linked to the “vendor”
+   resource.
+-  Removes useless functions in the order line ressource.
```
Comment (mmo) [inline RELEASE-NOTES.rst:118]: "order line"

File: RELEASE-NOTES.rst [hunk 14]
```diff
@@ -18,6 +18,226 @@
...
+Fixtures
+~~~~~~~~
+
+-  Adds patron transaction and patron transaction event fixtures.
+-  Adds loan fixtures to create active and paid overdue fees.
+
+Tests
+~~~~~
+
+-  Fixes issue with the daylight saving time timezone that occurs twice
```
Comment (mmo) [inline RELEASE-NOTES.rst:138]: daylight saving timezone

File: RELEASE-NOTES.rst [hunk 15]
```diff
@@ -18,6 +18,226 @@
...
+   focus form the input field was lost, forcing the librarian to click
+   to set the focus for the next validation.
+-  `#804 <https://github.com/rero/rero-ils/issue/804>`__: Example had to
+   be added in the patron editor to help the end user.
+-  `#826 <https://github.com/rero/rero-ils/issue/826>`__: In the checkin
+   interface, when a checkin item has a request, the name of the patron
+   that placed the request was not displayed in the correct order (last
+   name, first name).
+-  `#827 <https://github.com/rero/rero-ils/issue/827>`__: The alignment
+   of component had to be improved in the circulation interface. Items
```
Comment (mmo) [inline RELEASE-NOTES.rst:229]: The component alignment in the circulation interface had to be improved.

File: RELEASE-NOTES.rst [hunk 16]
```diff
@@ -18,6 +18,226 @@
...
+   with an action button were shorter than items without any button.
+-  `#829 <https://github.com/rero/rero-ils/issue/829>`__: Some flash
+   messages were missing when the librarian in checkin in items that has
```
Comment (mmo) [inline RELEASE-NOTES.rst:232]: *when the librarian in checkin in items that has requests* [something's wrong here...]

File: RELEASE-NOTES.rst [hunk 17]
```diff
@@ -18,6 +18,226 @@
...
+   requests, or fees, or that should be sent in transit.
+-  `#830 <https://github.com/rero/rero-ils/issue/830>`__: In the
+   circulation interface, some pickup location name had an extra
```
Comment (mmo) [inline RELEASE-NOTES.rst:235]: the name of some pickup locations had an extra

File: RELEASE-NOTES.rst (hunk 16 above)
Comment (iGormilhit) [reply RELEASE-NOTES.rst:232]: Yes, it should be "when the librarian **is** checkin in items that have requests".

**Pull request merged**